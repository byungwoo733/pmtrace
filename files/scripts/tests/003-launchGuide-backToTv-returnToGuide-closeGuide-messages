2016-05-10T07:50:29.499158Z [21978.105688729] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21978.105,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":1} 
2016-05-10T07:50:29.640688Z [21978.247211229] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21978.246,"PerfType":"Event","PerfGroup":"Input","key_code":1198,"key_value":1} 
2016-05-10T07:50:29.640760Z [21978.247282312] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21978.247,"PerfType":"Event","PerfGroup":"Input","key_code":1198,"key_value":0} 
2016-05-10T07:50:29.640830Z [21978.247351854] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21978.247,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":0} 
2016-05-10T07:50:29.668182Z [21978.274717145] user.info surface-manager-starfish [] qmlPerformance HOME_LAUNCHER_SHOWN {     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM",     "TIME": 22,     "UNIT": "ms" }  
2016-05-10T07:50:29.900493Z [21978.507027604] user.info surface-manager-starfish [] LSM HOME_WEDGE_SHOWN {     "CLOCK": 21978.506000000001,     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM" }  
2016-05-10T07:50:34.799598Z [21983.406120356] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21983.405,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":1} 
2016-05-10T07:50:34.928003Z [21983.534534898] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21983.534,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":0} 
2016-05-10T07:50:34.933791Z [21983.540313439] user.info sam [] SAM APP_LAUNCH {"CLOCK":21983.540,"app_id":"com.webos.app.tvuserguide","status":"start_prelaunching","PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:34.933997Z [21983.540519314] user.info sam [] SAM APP_LAUNCH {"CLOCK":21983.540,"app_id":"com.webos.app.tvuserguide","status":"start_memory_checking","PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:34.936392Z [21983.542914481] user.info sam [] SAM APP_LAUNCH {"CLOCK":21983.542,"app_id":"com.webos.app.tvuserguide","status":"start_launching","PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:34.937202Z [21983.543725273] user.info WebAppMgr [] WAM APPLAUNCH_START {"CLOCK":21983.543,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide","APP_ID":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:35.116143Z [21983.722666231] user.info WebAppMgr [] WAM APPLOADED {"CLOCK":21983.722,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide","APP_ID":"com.webos.app.tvuserguide","URL":"file:///usr/palm/applications/com.webos.app.tvuserguide/index.html","PID":11798} 
2016-05-10T07:50:35.839615Z [21984.446138232] user.info WebAppMgr [] WAM APPLOADED {"CLOCK":21984.445,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide","APP_ID":"com.webos.app.tvuserguide","URL":"file:///usr/palm/applications/com.webos.app.tvuserguide/Apps/k_kor/index.html","PID":11798} 
2016-05-10T07:50:36.202794Z [21984.809316607] user.info sam [] SAM GET_FOREGROUND_APPINFO {"CLOCK":21984.809,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:36.214004Z [21984.820525982] user.info WebAppMgr [] WAM WINDOW_FOCUSIN {"CLOCK":21984.820,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide","APP_ID":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:38.698197Z [21987.304720108] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21987.304,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":1} 
2016-05-10T07:50:38.879539Z [21987.486071108] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21987.485,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":0} 
2016-05-10T07:50:38.900823Z [21987.507354441] user.info surface-manager-starfish [] qmlPerformance HOME_LAUNCHER_SHOWN {     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM",     "TIME": 20,     "UNIT": "ms" }  
2016-05-10T07:50:39.143604Z [21987.750136441] user.info surface-manager-starfish [] LSM HOME_WEDGE_SHOWN {     "CLOCK": 21987.749,     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM" }  
2016-05-10T07:50:40.419639Z [21989.026161400] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21989.025,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":1} 
2016-05-10T07:50:40.579671Z [21989.186201776] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21989.185,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":0} 
2016-05-10T07:50:40.628655Z [21989.235176984] user.info sam [] SAM GET_FOREGROUND_APPINFO {"CLOCK":21989.233,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.livetv"} 
2016-05-10T07:50:42.089724Z [21990.696247318] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21990.695,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":1} 
2016-05-10T07:50:42.269599Z [21990.876130318] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21990.875,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":0} 
2016-05-10T07:50:42.318212Z [21990.924746735] user.info surface-manager-starfish [] qmlPerformance HOME_LAUNCHER_SHOWN {     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM",     "TIME": 19,     "UNIT": "ms" }  
2016-05-10T07:50:42.516712Z [21991.123246151] user.info surface-manager-starfish [] LSM HOME_WEDGE_SHOWN {     "CLOCK": 21991.121999999999,     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM" }  
2016-05-10T07:50:43.589755Z [21992.196276985] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21992.195,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":1} 
2016-05-10T07:50:43.749809Z [21992.356341860] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21992.355,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":0} 
2016-05-10T07:50:43.782100Z [21992.388622902] user.info sam [] SAM GET_FOREGROUND_APPINFO {"CLOCK":21992.388,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:44.330195Z [21992.936730527] user.info WebAppMgr [] WAM WINDOW_FOCUSIN {"CLOCK":21992.933,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide","APP_ID":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:45.209669Z [21993.816200111] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21993.815,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":1} 
2016-05-10T07:50:45.869184Z [21994.475716986] user.info surface-manager-starfish [] LSM HOME_WEDGE_SHOWN {     "CLOCK": 21994.474999999999,     "PerfGroup": "HomeScreenShown",     "PerfType": "LSM" }  
2016-05-10T07:50:46.489860Z [21995.096390903] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21995.096,"PerfType":"Event","PerfGroup":"Input","key_code":773,"key_value":0} 
2016-05-10T07:50:48.359878Z [21996.966401154] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21996.966,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":1} 
2016-05-10T07:50:48.518681Z [21997.125212696] user.info surface-manager-starfish [] IM IM_KEY_INPUT {"CLOCK":21997.124,"PerfType":"Event","PerfGroup":"Input","key_code":272,"key_value":0} 
2016-05-10T07:50:48.844098Z [21997.450630488] user.info WebAppMgr [] WAM WINDOW_FOCUSIN {"CLOCK":21997.450,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.tvuserguide","APP_ID":"com.webos.app.tvuserguide"} 
2016-05-10T07:50:48.924199Z [21997.530723113] user.info sam [] SAM GET_FOREGROUND_APPINFO {"CLOCK":21997.529,"PerfType":"AppLaunch","PerfGroup":"com.webos.app.livetv"}
