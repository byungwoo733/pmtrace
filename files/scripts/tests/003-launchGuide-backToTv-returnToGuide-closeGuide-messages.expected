Type: LSM
Group: HomeScreenShown
Start time: 21978.25
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:773
surface-manager-starfish       HOME_LAUNCHER_SHOWN       0.028    +0.028    TIME:22 UNIT:ms
surface-manager-starfish       HOME_WEDGE_SHOWN          0.259    +0.231    
Elapsed time (s) : 0.259

Type: AppLaunch
Group: com.webos.app.tvuserguide
Start time: 21983.53
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:272
sam                            APP_LAUNCH                0.006    +0.006    status:start_prelaunching
sam                            APP_LAUNCH                0.006    +0.0      status:start_memory_checking
sam                            APP_LAUNCH                0.008    +0.002    status:start_launching
WebAppMgr                      APPLAUNCH_START           0.009    +0.001    APP_ID:com.webos.app.tvuserguide
WebAppMgr                      APPLOADED                 0.188    +0.179    APP_ID:com.webos.app.tvuserguide URL:file:///usr/palm/applications/com.webos.app.tvuserguide/index.html PID:11798
WebAppMgr                      APPLOADED                 0.911    +0.723    APP_ID:com.webos.app.tvuserguide URL:file:///usr/palm/applications/com.webos.app.tvuserguide/Apps/k_kor/index.html PID:11798
sam                            GET_FOREGROUND_APPINFO    1.275    +0.364    
WebAppMgr                      WINDOW_FOCUSIN            1.286    +0.011    APP_ID:com.webos.app.tvuserguide
Elapsed time (s) : 1.286

Type: LSM
Group: HomeScreenShown
Start time: 21987.49
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:773
surface-manager-starfish       HOME_LAUNCHER_SHOWN       0.022    +0.022    TIME:20 UNIT:ms
surface-manager-starfish       HOME_WEDGE_SHOWN          0.264    +0.242    
Elapsed time (s) : 0.264

Type: LSM
Group: HomeScreenShown
Start time: 21990.88
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:773
surface-manager-starfish       HOME_LAUNCHER_SHOWN       0.05     +0.05     TIME:19 UNIT:ms
surface-manager-starfish       HOME_WEDGE_SHOWN          0.247    +0.197    
Elapsed time (s) : 0.247

Type: AppLaunch
Group: com.webos.app.tvuserguide
Start time: 21992.35
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:272
sam                            GET_FOREGROUND_APPINFO    0.033    +0.033    
WebAppMgr                      WINDOW_FOCUSIN            0.578    +0.545    APP_ID:com.webos.app.tvuserguide
Elapsed time (s) : 0.578

Type: LSM
Group: HomeScreenShown
Start time: 21993.81
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:1 key_code:773
surface-manager-starfish       HOME_WEDGE_SHOWN          0.66     +0.66     
Elapsed time (s) : 0.660

Type: AppLaunch
Group: com.webos.app.livetv
Start time: 21997.12
Process                        MsgID                     Time(s)  +Diff(s)  Extra
surface-manager-starfish       IM_KEY_INPUT              0.0      +0.0      key_value:0 key_code:272
WebAppMgr                      WINDOW_FOCUSIN            0.326    +0.326    APP_ID:com.webos.app.tvuserguide
sam                            GET_FOREGROUND_APPINFO    0.405    +0.079    
Elapsed time (s) : 0.405

